core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/EntityApiService.scala

```diff
@@ -92,6 +92,13 @@ trait EntityApiService extends UserInfoDirectives {
             }
           }
         } ~
+        path("workspaces" / Segment / Segment / "entities" / Segment / Segment / Segment ) { (workspaceNamespace, workspaceName, entityType, entityAttributeNamespace, entityAttributeName) =>
+          delete {
+            complete {
+              entityServiceConstructor(userInfo).deleteEntityAttribute(WorkspaceName(workspaceNamespace, workspaceName), entityType, entityAttributeNamespace, entityAttributeName, None, None)
+            }
+          }
+        } ~
```

Comment:
two comments on the API signature:
1) Instead of separate fields for the attribute namespace and attribute name, could you accept a delimited "namespace:name" string, and parse it via [`AttributeName.fromDelimitedName`](https://github.com/broadinstitute/rawls/blob/7cbf0de410a5deaf70bcdd89f0f0f1c5fe7fd7c0/model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala#L103)? This would allow the common case of the default namespace to omit the namespace, e.g. "myattribute" and only require specifying a namespace for non-default, e.g. "pfb:projectname".
2) Could this signature be modified to allow deleting multiple columns in one API call? It feels somewhat limiting to support only one at a time.

Reply:
I've made this delete multiple columns - but one thing I want to decide is the behavior if the some of the items in the list of column names are not found, but some are - what's the behavior? The way it's written now, the deletable columns will be deleted, while the columns that cannot be found will not, and a 204 is returned. It could instead make it fail all deletes if one delete fails.